content/en/tests/early_flake_detection.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Early Flake Detection
+kind: documentation
+description: Detect flakiness before it reaches the default branch
+aliases:
+- /continuous_integration/tests/early_flake_detection/
+further_reading:
+  - link: "/tests"
+    tag: "Documentation"
+    text: "Learn about Test Visibility"
+  - link: "/quality_gates"
+    tag: "Documentation"
+    text: "Learn about Quality Gates"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Early Flake Detection is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="#" btn_hidden="true" >}}
+Early Flake Detection is in public beta.
+{{< /callout >}}
+
+## Overview
+
+Early Flake Detection is Datadog's mechanism to tackle test flakiness. By identifying tests that are newly added and running them multiple times, Datadog can detect flakiness before it reaches the default branch. A study shows that up to [75% of flaky tests][1] can be identified this way.
+
+## How it works
+
+* Datadog's backend automatically stores unique tests for a given test service. These are **known tests**.
+* Before a test session runs, the Datadog library fetches the list of known tests.
+* If a test is in this list, it's executed normally.
+* If a test is **not** in this list, the Datadog library automatically retries it up to 10 times.
+
+Running a test multiple times helps detecting potential issues. If any of the test attempts fails, Datadog automatically tags it as flaky. You may then choose to block the merge of the feature branch with a [Quality Gate][2].
+
+### Why does running a test multiple times help detect issues?
+
+A [flaky test][3] is a test that exhibits both a passing and failing status across multiple test runs for the same commit. This behavior is normally caused by some random component, such as a race condition caused by an unexpected drop in network speed. Running the test multiple times increases the likelihood of this random component to show up.
```

Comment:
I don't understand well enough what "exhibits both a passing and a failing status" means, maybe we could rephrase it a bit easier? Like, "test that fails and passes for the same commit" or "test that randomly fails after some number of retries".

Reply:
this comes directly from our docs: https://docs.datadoghq.com/glossary/#flaky-test but I agree it's not great. I'll think of something better

Reply:
another pass in https://github.com/DataDog/documentation/pull/22556/commits/4577adb7496add2f8e7f01b72f91924808c02e8d